store_test.go

```diff
@@ -6,16 +6,23 @@ import (
 	"testing"
 
 	"github.com/containers/storage/pkg/idtools"
+	"github.com/containers/storage/pkg/reexec"
 	digest "github.com/opencontainers/go-digest"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
+func newTestStore(t *testing.T, options StoreOptions) Store {
+	store, err := GetStore(options)
```

Comment:
Creating this helper would primarily be beneficial if most of the `StoreOptions` (set up to make the tests runnable unprivileged on macOS, and the like) were set here. Deduplicating 2 lines but still having 15 lines copy&pasted is not that much of an improvement; removing 15 lines, and having the individual tests ~not care about how to deal with VFS and ID mapping, would be more useful.

I.e.
```go
   store := newTestStore(t, StoreOptions{PullOptions: pullOpts}) // in the caller which tests pullOpts

func newTestStore(… testOptions) {
   options := testOptions // A shallow copy
   options.RunRoot = …
   … // 15 more lines
   … GetStore…
}
```